react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,23 @@
+import logo from './logo.svg';
+import './App.css';
+import CustomButton from './CustomButton';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <div>
+        <CustomButton color={'red'} text={'i'} />
```

Comment:
If you are passing text, you can specify the props with just `"` instead of using `{}`:
`<CustomButton color="red" text="i" />`